platform/android/java_godot_io_wrapper.cpp

```diff
@@ -157,6 +158,15 @@ void GodotIOJavaWrapper::set_screen_orientation(int p_orient) {
 	}
 }
 
+int GodotIOJavaWrapper::get_screen_orientation() const {
+	if (_get_screen_orientation) {
+		JNIEnv *env = ThreadAndroid::get_env();
+		return env->CallIntMethod(godot_io_instance, _get_screen_orientation);
+	} else {
+		return -1;
```

Comment:
Might be safer to return `0` here. While incorrect, it would ensure that the value matches the [`OS::ScreenOrientation` enum](https://github.com/godotengine/godot/blob/231e07504893d684f674162f2237bdc81358b74e/core/os/os.h#L464).